pyproject.toml

```diff
@@ -111,7 +112,34 @@ exclude = [
 ]
 extraPaths = ["./apps/prairielearn/python"]
 pythonVersion = "3.10"
+executionEnvironments = [
```

Comment:
Until https://github.com/tamasfe/taplo/issues/332 happens, there isn't a great syntax for splitting this up.

Reply:
Worth adding a todo mentioning this?